suite-native/notifications/src/components/TransactionNotification.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Box, Text } from '@suite-native/atoms';
+import { AccountKey, TransactionType } from '@suite-common/wallet-types';
+import { TransactionIcon } from '@suite-native/transactions';
+import { CryptoAmountFormatter, AccountAddressFormatter } from '@suite-native/formatters';
+import {
+    TransactionsRootState,
+    selectTransactionByTxidAndAccountKey,
+    selectTransactionFirstTargetAddress,
+} from '@suite-common/wallet-core';
+import {
+    RootStackRoutes,
+    StackNavigationProps,
+    RootStackParamList,
+} from '@suite-native/navigation';
+import { Icon } from '@trezor/icons';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles/src';
+
+import { AnnouncementNotification } from './AnnouncementNotification';
+
+type TransactionNotificationProps = {
+    txid: string;
+    accountKey: AccountKey;
+    isHiddenAutomatically?: boolean;
+};
+
+type TransactionNotificationDescriptionProps = {
+    amount: string | number;
+    transactionType: TransactionType;
+    networkSymbol: NetworkSymbol;
+    targetAddress?: string;
+};
+
+type TransactionTypeProperties = {
+    title?: string;
+    preposition?: string;
+};
+
+const addressContainerStyle = prepareNativeStyle(_ => ({
+    maxWidth: '35%',
+}));
+
+const transactionTypeToContentMap = {
+    recv: { title: 'Incoming transaction', preposition: 'from' },
+    sent: { title: 'Sending transaction', preposition: 'to' },
+    self: { title: 'Sending transaction', preposition: 'to' },
+    joint: { title: 'Sending transaction', preposition: 'to' },
+    failed: { title: 'Sending transaction', preposition: 'to' },
+    unknown: { title: 'Sending transaction', preposition: 'to' },
+} as const satisfies Record<TransactionType, TransactionTypeProperties>;
+
+export const TransactionNotificationDescription = ({
+    amount,
+    targetAddress,
+    transactionType,
+    networkSymbol,
+}: TransactionNotificationDescriptionProps) => {
+    const { applyStyle } = useNativeStyles();
+    const { preposition } = transactionTypeToContentMap[transactionType];
+    return (
+        <Box flexDirection="row">
+            <CryptoAmountFormatter
+                value={amount}
+                network={networkSymbol}
+                isBalance={false}
+                color="gray500"
+                variant="label"
+            />
+            <Text color="gray500" variant="label">
+                {` ${preposition} `}
```

Comment:
Why not just `{preposition}`?

Reply:
because there would be no whitespace around. The result would look like e.g `0.003 BTCfromdsjf...fskdfd`

Reply:
I was going to say to use marginHorizontal, but I guess it's easier like this